HandsOn/Group11/mappings/RML_operations-with-links.rml

```diff
@@ -0,0 +1,169 @@
+    rml:source <https://drive.google.com/file/d/12ZgCUhrQx0pGQN3jjTdLPUmvOPbJP0gQ/view?usp=share_link>;
+    rml:referenceFormulation ql:CSV;
+    rml:iterator "csvRow";
+];
+
+
+    rr:subjectMap [
+        rr:template "http://artwork.org/artwork/{id}";
+        rr:termType rr:IRI;
+        rr:class art:Artwork;
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasId ;
+        rr:objectMap [
+            rr:reference "id";
+            rr:datatype xsd:integer;
+        ]
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasAccessionNumber ;
+        rr:object [
+            rr:reference "accession_number";
+            rr:datatype xsd:string;
+             ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasArtist ;
+        rr:object [
+            rr:reference "artist" ;
+            rr:datatype xsd:string;
+            ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasArtistRole ;
+        rr:object [
+            rr:reference "artistRole";
+            rr:datatype xsd:string;
+             ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasArtistId ;
+        rr:object [
+            rr:reference "artistId";
+            rr:datatype xsd:interger; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasTitle ;
+        rr:object [
+            rr:reference "title";
+            rr:datatype xsd:string; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasMedium ;
+        rr:object [
+            rr:reference "medium";
+            rr:datatype xsd:string; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasCreditLine ;
+        rr:object [
+            rr:reference "creditLine";
+            rr:datatype xsd:string; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art::madeInYear ;
+        rr:object [
+            rr:reference "year";
+            rr:datatype xsd:integer; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art::acquiredInYear ;
+        rr:object [
+            rr:reference "acquisitionYear";
+            rr:datatype xsd:integer; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasThumbnailCopyright ;
+        rr:object [
+            rr:reference "thumbnailCopyright";
+            rr:datatype xsd:string; ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasThumbnailUrl ;
+        rr:object [
+            rr:reference "thumbnailUrl";
+            rr:datatype xsd:anyURI;
+         ];
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasUrl ;
+        rr:object [
+            rr:reference "url";
+            rr:datatype xsd:anyURI; ];
+    ].
+
+    rr:predicateObjectMap [
+        rr:predicate art:hasDimension ;
+        rr:object [
+            rr:reference "http://artwork.org/artwork/{id}/dimensions"; ];
+    ].
+
+    rr:predicateObjectMap [
+        rr:predicate art:sameAsDbpediaArtist ;

```

Comment:
This should actually be predicateMap where you set it to be owl:sameAs property. Making it the way you made it we would get linked data where each artist object has string property - this is not a reference to other data. Making it predicateMap we'll get owl:sameAs property for each artist and this is what we need.
See: https://moodle.upm.es/titulaciones/oficiales/pluginfile.php/11170952/mod_resource/content/12/MGLD_Linking_DG.pdf
Slide 27